element-connector/pom.xml

```diff
@@ -21,6 +21,11 @@
 			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.netty</groupId>
+			<artifactId>netty-all</artifactId>
+			<version>4.0.28.Final</version>
```

Comment:
Is there a particular requirement to use 4.0.28? We should tak care to use a version that has already undergone the Eclipse IP review process (which 4.0.28 has), but we can e.g. also use a more recent version like 4.1.0 or 4.0.36 ...
any preference?